packages/thumbnail-grid-block/src/settings.ts

```diff
@@ -0,0 +1,259 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    IconEnum,
+    appendUnit,
+    defineSettings,
+    maximumNumericalOrPercentOrAutoRule,
+    maximumNumericalOrPixelOrAutoRule,
+    minimumNumericalOrPercentRule,
+    minimumNumericalOrPixelOrAutoRule,
+    numericalOrPercentRule,
+    numericalOrPixelRule,
+    presetCustomValue,
+} from '@frontify/guideline-blocks-settings';
+import { getBackgroundSettings, getBorderRadiusSettings, getBorderSettings } from '@frontify/guideline-blocks-shared';
+import { CaptionPosition, GutterValues, HorizontalAlignment, VerticalAlignment } from './types';
+
+const IMAGE_POSITION_CHOICE_ID = 'imagePosition';
+const GUTTER_CHOICE_ID = 'gutterPreset';
+const GUTTER_CUSTOM_ID = 'customGutter';
+
+const widthMap: Record<string, string> = {
+    '25%': '25%',
+    '33%': '33%',
+    '50%': '50%',
+    '75%': '75%',
+};
+
+const gutterMap: Record<GutterValues, string> = {
+    [GutterValues.Auto]: '',
+    [GutterValues.Small]: '10px',
+    [GutterValues.Medium]: '30px',
+    [GutterValues.Large]: '50px',
+};
+
+export const settings = defineSettings({
+    layout: [
+        {
+            id: 'styleSection',
+            type: 'sectionHeading',
+            label: 'Block',
+            blocks: [
+                {
+                    id: 'columnCount',
+                    label: 'Columns',
+                    type: 'slider',
+                    defaultValue: '3',
+                    info: 'Sets the number of columns on large screens. Columns will break automatically on smaller screens.',
+                    choices: [
+                        {
+                            value: '1',
+                            label: '1',
+                        },
+                        {
+                            value: '2',
+                            label: '2',
+                        },
+                        {
+                            value: '3',
+                            label: '3',
+                        },
+                        {
+                            value: '4',
+                            label: '4',
+                        },
+                        {
+                            value: '5',
+                            label: '5',
+                        },
+                    ],
+                },
+                {
+                    id: 'hasCustomGutter',
```

Comment:
You could use the gutter settings from the shared folder.

Reply:
oh true, thanks! done